scons: building terminated because of errors.

<!--
README: Incompatibilities and broken features in the current master branch / 3.0-alpha
are known and expected due to important refactoring work, so no need to report them for now. Thanks!
-->
**Operating system or device - Godot version:**
Linux 4.4.39-1-MANJARO x86_64

**Issue description:**
<!-- What happened, and what was expected. -->
/d/libs/godot/core/reference.h:298: undefined reference to `ResourceImporterTexture::ResourceImporterTexture()'


**Steps to reproduce:**
g++ -o bin/godot.x11.tools.64 platform/x11/godot_x11.x11.tools.64.o platform/x11/context_gl_x11.x11.tools.64.o platform/x11/os_x11.x11.tools.64.o platform/x11/key_mapping_x11.x11.tools.64.o platform/x11/joypad_linux.x11.tools.64.o main/libmain.x11.tools.64.a main/tests/libtests.x11.tools.64.a modules/libmodules.x11.tools.64.a drivers/libdrivers.x11.tools.64.a tools/libtool.x11.tools.64.a scene/libscene.x11.tools.64.a servers/libservers.x11.tools.64.a core/libcore.x11.tools.64.a -lX11 -lXinerama -lXcursor -lXrandr -lssl -lcrypto -lfreetype -lpng16 -lasound -lpulse-simple -lpulse -lz -lGL -lpthread -ldl


**Link to minimal example project:**
<!-- Optional but very welcome. You can drag and drop a zip archive to upload it. -->:

--- a/tools/editor/import/SCsub
+++ b/tools/editor/import/SCsub
@@ -1,0 +1,5 @@
+#!/usr/bin/env python
+
+Import('env')
+Export('env')
+env.add_source_files(env.tool_sources, "*.cpp")

